src/devices/src/virtio/block/block_test_utils.rs

```diff
@@ -0,0 +1,39 @@
+pub mod test_utils {
```

Comment:
With current code you have the following hierarchy:
`devices::virtio::block::block_test_utils::test_utils`

You can simplify this by renaming the file from `block_test_utils.rs` to `test_utils.rs` since using a sibling file next to `mod.rs` implies all code in that file being in a module with the same name as the filename.

The hierarchy would then become: `devices::virtio::block::test_utils` which is simpler and nicer.

Same for all others.